examples/nas/benchmarks/nasbench101.requirements.txt

```diff
@@ -1,4 +1,4 @@
-tensorflow==1.15.2
+tensorflow
```

Comment:
Leave a comment here to state that we've tested on v1.15.2.

Reply:
done